### ISSUE TITLE
Worker is not defined

### ISSUE THREAD

**Issue by geoffreywiseman** [Rebilly/ReDoc#485]
Testing redoc-cli on an OpenAPI 3.0.1 YAML:

```
± npx redoc-cli bundle ../openapi.yml
Mobx observer: You are using 'observer' on React.PureComponent. These two achieve two opposite goals and should not be used together
Prerendering docs
Worker is not defined
```

Near as I can tell `redox-cli serve` works fine, but `bundle` isn't behaving for me.

**Comment by skyjia** [comment]
+1

**Comment by RomanGotsiy** [comment]
Fixed in redoc@2.0.0-alpha.20 and redoc-cli@0.3.8

**Comment by geoffreywiseman** [comment]
Yup, I can confirm that `redoc-cli@0.3.8` works on the same file that it was failing on before.
